distr_test/tests/skew_normal.rs

```diff
@@ -257,6 +257,9 @@ fn owen_t(h: f64, a: f64) -> f64 {
 }
 
 fn normal_cdf(x: f64, mean: f64, std_dev: f64) -> f64 {
+    // f64::erfc from feature `float_erf` may be used over special::Primitive::erfc
```

Comment:
Note that this is a reproducibility hazard.

Reply:
It is, but if I add `#![feature(float_erf)]` to the module then the test still passes (with a warning that `special::Primitive` is unused).